bcauth/jinja2/oauth2_provider/application_form.html

```diff
@@ -0,0 +1,47 @@
+{% extends "account/profile.html" %}
+
+{% block head_subtitle %} - {{ _("OAuth2 Client Application") }} - {{ _("Edit") }}{% endblock %}
+{% block body_title %}{{ _("OAuth2 Client Application") }} - {{ _("Edit") }}{% endblock %}
+
+{% block content %}
+
+<form method="post" action="{% block app_form_action_url %}{{ url('oauth2_provider:update', application.id) }}{% endblock app_form_action_url %}">
+  <h3>
+      {%- block app_form_title -%}
+      {{ _("Edit Application %(app_name)s")|format(app_name=application.name) }}
+      {%- endblock app_form_title -%}
+  </h3>
+  {{ csrf_input }}
+
+  {% for field in form %}
+    <div class="form-group {% if field.errors %}has-error{% endif %}">
+      <label class="control-label" for="{{ field.id_for_label }}">{{ field.label }}</label>
+
+        <div class="controls">
+          {{ field }}
+          {% for error in field.errors %}
+            <span class="help-inline">{{ error }}</span>
+          {% endfor %}
+          {% if field.help_text %}<span id="{{ field.id_for_label + 'HelpBlock'}}" class='help-block'>{{ field.help_text }}</span>{% endif %}
```

Comment:
Minor nits: I'd add a space between `'` and `}}` and also switch the `'` for the class value to `"`.